src/fe/fe_lagrange_shape_2D.C

```diff
@@ -44,6 +44,7 @@ Real FE<2,LAGRANGE>::shape(const ElemType type,
         switch (type)
           {
           case QUAD4:
+          case QUADSHELL4:
```

Comment:
I suppose we should also technically support the new geometric element types for all the FEs (hierarchic, monomial, etc.) that currently support TRI3 and QUAD4, since they are basically the same thing?